Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => {
+	return ( 
+		<div className="cardP">
+			<h1>Get started with React</h1>
+            <p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>
+            <p className="org">reactjs.org</p>
+            <a href="https://reactjs.org"></a>
```

Comment:
you can add a `target="_blank"` so that clicking on the anchor will open a new tab